notifications.yaml

```diff
@@ -0,0 +1,268 @@
+swagger: '2.0'
+info:
+  version: 0.0.1
+  title: Notification API
+  description: Notification API.
+host: localhost
+basePath: /api/v1
+schemes:
+  - https
+paths:
+  '/messages/{id}':
+    parameters:
+      - name: id
+        in: path
+        type: string
+        required: true
+    get:
+      operationId: getMessage
+      summary: Get Message
+      description: Returns a previously submitted message.
+      responses:
+        '200':
+          description: Message found.
+          schema:
+            $ref: '#/definitions/AcceptedMessage'
```

Comment:
submitted message -> `SubmittedMessage`

or

accepted message -> `AcceptedMessage`

We should use the same terminology everywhere (in the model names, in the descriptions, in the state enum etc...)